### PR TITLE
Bienaleweb 69 add arrows

### PR DESCRIPTION
**Pull request by igor-kamil**
Screenshot 2024-04-18 at 13 44 12

### CODE REVIEW

File: resources/js/App.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
         class="container mx-auto px-4 md:px-0 pt-3 md:pt-6 text-blue-darker"
         :class="
             isMenuOpen
-                ? 'bg-blue-lighter flex flex-col w-screen h-screen fixed z-10 left-0 top-0'
+                ? 'bg-blue-lighter flex flex-col w-screen h-screen fixed z-50 left-0 top-0'
```
Comment (FrantisekMichalSebestyen) [inline resources/js/App.vue:13]:
```suggestion
                ? 'bg-blue-lighter flex flex-col w-screen h-screen fixed z-20 left-0 top-0'
```

File: resources/js/Detail.vue [hunk 2]
```diff
@@ -37,23 +37,13 @@
                     :to="{ name: 'tree', params: { id: state.indexPrev } }"
                     class="group bg-blue-lighter bg-opacity-80 absolute left-4 bottom-1/2 w-8 h-8 flex items-center justify-center blur-[0.5px] rounded-full"
                 >
-                    <svg xmlns="http://www.w3.org/2000/svg" class="w-6 h-6 stroke-blue-darkest" viewBox="0 0 256 256">
-                        <path
-                            stroke-width="16"
-                            d="M220,128a4,4,0,0,1-4,4H49.66l65.17,65.17a4,4,0,0,1-5.66,5.66l-72-72a4,4,0,0,1,0-5.66l72-72a4,4,0,0,1,5.66,5.66L49.66,124H216A4,4,0,0,1,220,128Z"
-                        ></path>
-                    </svg>
+                    <ArrowLeftIcon class="w-6 h-6" />
```
Comment (FrantisekMichalSebestyen) [inline resources/js/Detail.vue:40]: <3

File: resources/js/Home.vue [hunk 3]
```diff
@@ -138,6 +144,8 @@
 <script setup>
 import { ref, onMounted } from "vue";
 import { useRouter } from "vue-router";
+import ArrowLeftIcon from "./icons/ArrowLeftIcon.vue";
+import ArrowRightIcon from "./icons/ArrowRightIcon.vue";
 
 const router = useRouter();
 const arboretumContainer = ref(null);
```
Comment (FrantisekMichalSebestyen) [inline resources/js/Home.vue:151]: `const arboretumWrapper = ref(null)`

**Review by FrantisekMichalSebestyen — APPROVED** [review]
just fix, pls

File: resources/js/App.vue (hunk 1 above)
Comment (igor-kamil) [reply resources/js/App.vue:13]: this will make arrows still visible. I put `z-30`

File: resources/js/Home.vue (hunk 3 above)
Comment (igor-kamil) [reply resources/js/Home.vue:151]: I need both wrapper & container

**Pull request merged**